src/main/java/com/assignment1/userinterface/UserInterface.java

```diff
@@ -0,0 +1,79 @@
+package com.assignment1.userinterface;
+
+import com.assignment1.Type;
+import com.assignment1.exceptions.RuntimeExceptionCustom;
+import com.assignment1.item.ItemEntity;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Class to implement methods of user interface.
+ */
+@SuppressWarnings("PMD.SystemPrintln")
+public class UserInterface implements UserInterfaceHandler {
```

Comment:
as previously mentioned, this interface needs some thinking

Reply:
Removed this interface.